content/snippets/capacity-planning.md

```diff
@@ -0,0 +1,97 @@
+# Capacity Planning
+
+
+Even the most minimally functional `rippled` server must contain the most recently validated Ledger versions to submit transactions to the network and verify the integrity of the XRP Ledger. Beyond these, consider the following server requirements:
+
+- Handling ever-increasing transaction volume
+- Servicing transaction reporting information to clients
+- Maintaining varying amounts of historical data
+
+Capacity planning for `rippled` involves two factors:
+
+- The [configuration settings](#configuration-settings) that affect resource utilization
+- The [network and hardware] (#network and hardware) requirements to achieve consistent, good performance across the XRP Ledger network
+
+## Configuration Settings
+
+Ripple recommends the following guidelines to improve performance. You can set the following parameters in the `rippled.cfg` file to improve performance for your `rippled` server.
+
+### Node Size
+
+The `node_size` parameter determines the size of database caches. Ripple recommends you always set it to `huge` for production servers. Increased cache size requires less disk I/O and allows `rippled` to improve performance. The trade-off to improving performance better is that memory requirements increase.
+
+### Node DB Type
+
+The `type` field in the `node_db` section of the `rippled.cfg` file sets the type of key-value store that `rippled` uses to persist the XRP Ledger in the Ledger Store. You can set the value to either `rocksdb` or `nudb`. For setting the store that `rippled` uses to persist history shards, if you are participating in history sharding, see [History Sharding](XREF: concept-history-sharding.md#shard-store-configuration).
+
+#### RocksDB vs NuDB
+RocksDB requires approximately one-third less disk storage than NuDB and provides a corresponding improvement in I/O latency. However, this comes at a cost of increased memory utilization as storage size grows. NuDB, on the other hand, has nearly constant performance and memory footprint regardless of storage.
```

Comment:
Maybe add a link to the storage section from here or the "Node DB Type" section?